src/components/InlineSpinner/__snapshots__/InlineSpinner.spec.tsx.snap

```diff
@@ -7,16 +7,20 @@ exports[`InlineSpinner renders in a custom color 1`] = `
   width: 1.25rem;
   height: 1.25rem;
   vertical-align: text-bottom;
-  border: 0.125rem solid;
+  border: 0.125rem solid #069D4F;
```

Comment:
from where this color comes from? shouldn't it be `inherit`?

Reply:
This is the test to ensure the spinner accepts a custom color 🎨